activepieces/activepieces
# feat: added external links for docs

## What does this PR do?

Added external links to docs when click on a flow step, you will see it next to the step type name.

Fixes # (issue)
Addresses #421 

## Type of change



## How should this be tested?

Click on a flow step and click the external url icon, it should work on everything other than storage, since we don't have docs for that.
